js/index.js

```diff
@@ -1,36 +1,26 @@
-/* eslint no-use-before-define: ["error", {"functions":false}] */
+/* eslint-disable no-undef */
 
 const bookListView = document.getElementById('booklist');
 
-let bookList = JSON.parse(localStorage.getItem('books'));
-if (bookList == null) {
-  bookList = [];
-}
-
-function removeBook(book) {
-  bookList = bookList.filter(
-    (singleBook) => book.id !== singleBook.id,
-  );
-  localStorage.setItem('books', JSON.stringify(bookList));
-  initiew();
-}
-
 function initiew() {
   bookListView.innerHTML = '';
-  bookList.forEach((element) => {
+  const bookListManager = new BookListManger();
+  bookListManager.bookList.forEach((element) => {
```

Comment:
- [ ] According to the Project requirements you should use JavaScript `classes` instead of `objects` and `arrays`, please note that you can use objects and arrays but it has to be **inside** the classes. Please address this 👍🏻